lib/src/kbaserelationengine/search/ElasticIndexingStorage.java

```diff
@@ -211,7 +219,15 @@ public void indexObjects(String objectType, String objectName, long timestamp,
         doc.putAll(indexPart);
         doc.put("guid", id.toString());
         doc.put("otype", objectType);
-        doc.put("oname", objectName);
+
+        doc.put("oname", data.getName());
+        doc.put("creator", data.getCreator());
```

Comment:
I think you should add description for new fields to Elastic data structure definition code.

Reply:
Good catch, fixed. Ready for review again.